### DISCUSSION TITLE
Supported size of Models

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2056] [Q&A] [answered]
What is the supported size for model files? I tried to load the glTF file with the size of 40Mb and the page keeps crashing on iOS chrome and safari.

**Comment by elalish** [comment, marked as answer]
It's not something we control; it sounds like you ran out of GPU RAM, either for your device or how much a process is allowed. [KTX2 texture compression](https://modelviewer.dev/examples/loading/#ktx2Support) is one good way to improve this, though other mesh optimizations are possible too.
